docs/source/traits_user_manual/intro.rst

```diff
@@ -10,10 +10,8 @@ repository <http://github.com/enthought/traits>`_. Additional documentation
 for the Traits package is available,
 including:
 
-* *Traits API Reference*
-* `TraitsUI User Manual
-  <http://docs.enthought.com/traitsui/traitsui_user_manual/index.html>`_
-* Traits Technical Notes
+* `Traits API Reference <https://docs.enthought.com/traits/traits_api_reference/index.html>`_
+* `Traits Technical Notes <https://docs.enthought.com/traits/changelog.html>`_
```

Comment:
I don't know what this was originally supposed to refer to either; even going as far back as Traits 4.0, this wasn't linked to anything.

How about we just drop this entry? That then leaves us with the awkwardness of an itemised list with a single item, so maybe we should simply reword the sentence that introduces this list, and link to the API reference directly from it. Thoughts?

Reply:
SGTM